### PR TITLE
Mobile form

### PR DESCRIPTION
**Pull request by KUBAHO3**
In this project:

- I added the html5 forms with validation
- I added action to submit the form to the link provided
- I added responsive background vectors as an image
- I added to the left curve/radius as from Figma UI
- I added button hoovering on the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,7 +180,33 @@ <h4 class="skill-name">Skills</h4>
...
+            <h3 class="contact-title">
+                Contact me
+            </h3>
+            <p class="contact-explanation">
+                If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+            </p>
+            <form action="https://formspree.io/f/xrgvqlpj" method="post">
+                <div>
+                    <label for="Name" hidden>Name</label>
+                    <input id="username" type="text" name="username" required pattern="^[a-zA-Z]{3,30}$" title="Name should be text and not more than 30 characters" placeholder="Enter your Name">
```
Comment (zieeco) [inline index.html]: - Hey, please, let's kindly add the missing max length of 30 characters to meet this part of the project requirement.
- Kindly remove the pattern you've added as this is preventing your form from being submitted.

File: index.html [hunk 2]
```diff
@@ -180,7 +180,33 @@ <h4 class="skill-name">Skills</h4>
...
+                </div>
+                <div>
+                    <label for="email" hidden>Email</label>
+                    <input id="useremail" type="email" name="email" required pattern="^([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9_\-\.]+)\.([a-zA-Z]{2,5})$" title="Only emails are allowed here" placeholder="Enter your Email">
+                </div>
+                <div>
+                    <label for="email" hidden>Description</label>
+                    <textarea id="description" type="text" name="description" required pattern="^[a-zA-Z0-9]{1,500}$" title="description should be not more than 500 characters" placeholder="Write your message here"></textarea>
```
Comment (zieeco) [inline index.html]: - Hey, let's also add the missing max length of 500 characters to meet this part of the project requirement.

File: index.html [hunk 3]
```diff
@@ -180,7 +180,33 @@ <h4 class="skill-name">Skills</h4>
...
+                If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+            </p>
+            <form action="https://formspree.io/f/xrgvqlpj" method="post">
+                <div>
+                    <label for="Name" hidden>Name</label>
+                    <input id="username" type="text" name="username" required pattern="^[a-zA-Z]{3,30}$" title="Name should be text and not more than 30 characters" placeholder="Enter your Name">
+                </div>
+                <div>
+                    <label for="email" hidden>Email</label>
+                    <input id="useremail" type="email" name="email" required pattern="^([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9_\-\.]+)\.([a-zA-Z]{2,5})$" title="Only emails are allowed here" placeholder="Enter your Email">
```
Comment (zieeco) [inline index.html]: - Hi, please, kindly remove the pattern you've added as this is preventing your form from being submitted.

File: style.css [hunk 4]
```diff
@@ -219,3 +229,52 @@ p.description {
...
   padding-top: 15px;
 }
+
+.my-form {
+  padding-top: 10vh;
+  padding-bottom: 5vh;
+  background-image: url("./images/contactbg.png");
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: 100%;
```
Comment (zieeco) [inline style.css]: - [ ]  Hey, the background image is not showing completely and it's missing the black horizontal line at the bottom of the design as seen in the Figma template. please, let's fix this issue. see the images below:

*your design*
Screenshot from 2023-02-21 11-49-42

*Figma design*
Screenshot from 2023-02-21 11-50-33

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (KUBAHO3) [reply index.html]: Thank you for your comment! with due respect, I appreciate your suggestions but my form is successfully being submitted and the pattern can also play a role of maximum length! I will consider your request but not that the same functionality can be done in multiple ways
review1
review2

I hope you can see it from the above screenshot! Thanks

File: index.html (hunk 2 above)
Comment (KUBAHO3) [reply index.html]: When specified in pattern the max length can be defined from there

File: index.html (hunk 3 above)
Comment (KUBAHO3) [reply index.html]: As you have seen in the above screenshot the form is being submitted successfully

File: style.css (hunk 4 above)
Comment (KUBAHO3) [reply style.css]: Thank you Once again! the black bar above represents a smartphone view of the page and is defined by the mobile maker! Though I can add it to make my project approved but it really represents the home button on smartphone and figma designers add it for the purpose of clarity

**Comment by KUBAHO3** [comment]
Thank you for your review! I have made all changes required as mentioned above!

File: style.css (hunk 4 above)
Comment (OyePriscilla) [reply style.css]: - [ ] Let's work more on the placement of background-image as pointed earlier. You are almost there 👍 I think you need to reduce the background-size and position it to the right. 
```suggestion
  background-image: url("./images/contactbg.png");
  background-color: #6070ff;
  background-repeat: no-repeat;
  background-size: 55%;
  background-position: right 21px;
```

## Your Design
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @KUBAHO3 and @popey17 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
GitHub flow observed ✔️ 
Linters passed ✔️ 
PR summary ✔️ 
Specified HTML input max length attribute ✔️ 
Form submitted as expected ✔️ 

## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @KUBAHO3     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**